aodncore/util/wfs.py

```diff
@@ -8,11 +8,14 @@
 from ..util import IndexedSet
 
 __all__ = [
+    'DEFAULT_WFS_VERSION',
     'WfsBroker',
     'get_filter_for_file_url',
     'ogc_filter_to_string'
 ]
 
+DEFAULT_WFS_VERSION = '1.0.0'
```

Comment:
Shouldn't this be "1.1.0"?

Reply:
See my comment below. If we immediately updated it to 1.1.0, it would break aodndata until https://github.com/aodn/python-aodndata/pull/197 is merged/built/deployed.